client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -363,7 +366,9 @@ export class DomainDetailsForm extends PureComponent {
 	}
 
 	renderFaxField() {
-		return <Input label={ this.props.translate( 'Fax' ) } { ...this.getFieldProps( 'fax' ) } />;
+		return this.needsFax() ? (
```

Comment:
I think it's better to separate the concerns here. You should be able to just `this.needsFax() && renderFaxField()` in the caller.

Reply:
Fair enough. Agree that UI logic should be kept in the render method. Thanks!